server/scripts/populate-test-data/populateFakeTestResults.js

```diff
@@ -5,31 +5,49 @@ const {
 } = require('../../models/services/BrowserService');
 const { query, mutate } = require('../../tests/util/graphql-test-utilities');
 
+/**
+ *
+ * @param {number} testPlanRunId
+ * @param {string['completeAndPassing' |
+ * 'completeAndFailingDueToIncorrectAssertions' |
+ * 'completeAndFailingDueToNoOutputAssertions' |
+ * 'completeAndFailingDueToUnexpectedBehaviors' |
+ * 'completeAndFailingDueToMultiple' |
+ * 'incompleteAndEmpty' |
+ * 'incompleteAndPassing' |
+ * 'incompleteAndFailingDueToIncorrectAssertions' |
+ * 'incompleteAndFailingDueToNoOutputAssertions' |
+ * 'incompleteAndFailingDueToUnexpectedBehaviors' |
+ * 'incompleteAndFailingDueToMultiple']} fakeTestResultTypes
+ * @param {import('sequelize').Transaction} transaction
+ * @param {'SINGLE' | 'ALL' | number} numFakeTestResultConflicts
```

Comment:
This is a little interesting. Could you just do a number with `Math.Infinity` for 'ALL'? I guess it is a little strange to read 'infinity' when we really mean all. No strong feelings here but just struck me that this could be a single 
```js
testResult.scenarioResults
        .slice(0, numFakeTestResultConflicts)
        .forEach(setResult);
```

Reply:
Good point! Addressed in https://github.com/w3c/aria-at-app/pull/1224/commits/4b70b0b9d94a8ba55897dd40852f726229e6df4d